.github/workflows/lint.yaml

```diff
@@ -17,24 +17,27 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           # Full git history is needed to get a proper
           # list of changed files within `super-linter`
           fetch-depth: 0
 
       - name: Lint
-        uses: github/super-linter@v4
+        uses: github/super-linter@v6
         env:
           VALIDATE_ALL_CODEBASE: false
           DEFAULT_BRANCH: master
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VALIDATE_MARKDOWN: false
           # These tools are handled by pre-commit, so we don't need SuperLinter
           # to run them as well
+          VALIDATE_CHECKOV: false
```

Comment:
These got introduced with the version bump. One of them (CHECKOV) throws some interesting new warnings [about our IaC setup](https://github.com/ccao-data/data-architecture/actions/runs/10210408742/job/28250004495#step:4:229). Might be worth pursuing, fixing, and then re-enabling this linter. What do you think?

Reply:
Interesting, thanks for flagging this! On first glance, the [top-level permissions error](https://github.com/ccao-data/data-architecture/actions/runs/10210332289/job/28249802654#step:4:251) seems reasonable enough and I would support fixing it, but I'm finding myself struggling to understand the [workflow_dispatch inputs check](https://github.com/ccao-data/data-architecture/actions/runs/10210332289/job/28249802654#step:4:238). I'm down to file an issue to get a handle on these checks and enable the linter!